### PR TITLE
Change load command readType to default readType

### PR DESCRIPTION
**Pull request by jiacheliu3**
Seems the load command is using CACHE_PROMOTE no matter what

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java [hunk 1]
```diff
@@ -96,8 +98,11 @@ private void load(AlluxioURI filePath, boolean local) throws AlluxioException, I
         load(newPath, local);
       }
     } else {
+      ReadPType readType = mFsContext.getPathConf(filePath)
```
Comment (gpang) [inline shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java:101]: Will `getPathConf` incur an RPC each time?

**Comment by yuzhu** [comment]
actually, i think it makes sense to use CACHE_PROMOTE for load.. I mean, how would load work if the user specified something like NO_CACHE?

**Comment by jiacheliu3** [comment]
> actually, i think it makes sense to use CACHE_PROMOTE for load.. I mean, how would load work if the user specified something like NO_CACHE?

In that case we should hard code `CACHE` I think?

**Comment by yuzhu** [comment]
either CACHE or CACHE_PROMOTE should be fine.  I think when a user explicitly loads something, he/she probably expect the best performance for those files, so moving to top tier aka CACHE_PROMOTE makes sense.  @calvinjia any opinion on this?

**Comment by jiacheliu3** [comment]
> either CACHE or CACHE_PROMOTE should be fine. I think when a user explicitly loads something, he/she probably expect the best performance for those files, so moving to top tier aka CACHE_PROMOTE makes sense. @calvinjia any opinion on this?

In https://github.com/Alluxio/alluxio/commit/3762d75eb611f388895ad9bea0746b17afc7cdd1 the `CACHE` implementation is changed and we changed `readType.default` from `CACHE_PROMOTE` to `CACHE` to avoid moving blocks around while serving a read request. Changing to `CACHE` to be consistent with that.

File: shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java (hunk 1 above)
Comment (jiacheliu3) [reply shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java:101]: changed to hard code `CACHE` so this is avoided

**Comment by jiacheliu3** [comment]
Added commit https://github.com/jiacheliu3/alluxio/commit/81d1b758b6614138cab5eda0a63c3c62a27e5e7c but it's not yet reflected on this PR...

**Comment by jiacheliu3** [comment]
Somehow this PR does not work. Created https://github.com/Alluxio/alluxio/pull/13281 instead

**Pull request closed without merging**